docs/08_getting_started.md

```diff
@@ -36,6 +139,99 @@ public Response execute(@Nonnull Request request) throws IOException, ReCaptchaE
 }
 ```
 
+<details>
+<summary>simple implementation</summary>
```

Comment:
This looks like the `DownloaderTestImpl`, instead of copying it I would just link it. 
https://github.com/TeamNewPipe/NewPipeExtractor/blob/dev/extractor/src/test/java/org/schabi/newpipe/downloader/DownloaderTestImpl.java

Reply:
idk.. both solutions seems ok